types/web3/web3-tests.ts

```diff
@@ -16,3 +16,8 @@ const myContract = new web3.eth.Contract(
 myContract.options.from = "0x1234567890123456789012345678901234567891";
 myContract.options.gasPrice = "20000000000000";
 myContract.options.gas = 5000000;
+
+const weiStr = web3.utils.toWei("100", "gwei");
+weiStr.endsWith(weiStr);
+const weiBn = web3.utils.toWei(web3.utils.toBN("1"));
```

Comment:
I suggest
```js
const weiBn: BigNumber = web3.utils.toWei(web3.utils.toBN("1"));
```
instead of line 22-23. What do you say?

Reply:
That's awesome, I don't even know why I didn't do it this way in the first place. Great idea.